Terminal.Gui/Views/CheckBox.cs

```diff
@@ -83,11 +83,11 @@ public CheckBox (int x, int y, ustring s, bool is_checked) : base (new Rect (x,
 		/// <param name="is_checked"></param>
 		void SetInitialProperties (ustring s, bool is_checked)
 		{
-			charNullChecked = new Rune (Driver != null ? Driver.NullChecked : '?');
-			charChecked = new Rune (Driver != null ? Driver.Checked : '√');
-			charUnChecked = new Rune (Driver != null ? Driver.UnChecked : '╴');
+			charNullChecked = Glyphs.NullChecked;
+			charChecked = Glyphs.Checked;
+			charUnChecked = Glyphs.UnChecked;
 			Checked = is_checked;
-			HotKeySpecifier = new Rune ('_');
+			HotKeySpecifier = Glyphs.HotKeySpecifier;
```

Comment:
Is `HotKeySpecifier` really a Glyph?  

I thought it was more like an escape character that indicates to Terminal.Gui where to change color when rendering (and to pick up the next character as the shortcuts?).

I think if the user was to edit this to another symbol quite a lot of stuff might break in Terminal.Gui apps especially if the author just assumes that its `_` .

Reply:
If all the `_` are replaced with the `Glyphs.HotKeySpecifier` there aren't any issue. That will imply the users have to change their code to $"{Glyphs.HotKeySpecifier}whatvever". But isn't a big deal with the editors replacement tools.

Reply:
Ill remove it from Glyphs. 

I included it as a mental model test and forgot to remove it.